spec/index.bs

```diff
@@ -4855,6 +4876,13 @@ run the following steps:
 
 A {{GPUProgrammableStage}} describes the entry point in the user-provided
 {{GPUShaderModule}} that controls one of the programmable stages of a [=pipeline=].
+Entry point names compare by their code points and
+do not follow [=UAX15 Equivalences=] for [[!UnicodeVersion14|Unicode Version 14.0.0]].
+
+Note: User agents should consider issuing developer-visible warnings in most or all
```

Comment:
I'm not sure this is the right time for an implementation to check for collisions.
I would probably drop this note here.

Reply:
@dneto0 done, thanks!